bundles/org.openhab.binding.webexteams/src/main/java/org/openhab/binding/webexteams/internal/WebexTeamsHandler.java

```diff
@@ -0,0 +1,315 @@
+/**
+ * Copyright (c) 2010-2022 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.webexteams.internal;
+
+import static org.openhab.binding.webexteams.internal.WebexTeamsBindingConstants.*;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.eclipse.jetty.client.HttpClient;
+import org.openhab.binding.webexteams.WebexTeamsActions;
+import org.openhab.binding.webexteams.internal.api.Message;
+import org.openhab.binding.webexteams.internal.api.NotAuthenticatedException;
+import org.openhab.binding.webexteams.internal.api.Person;
+import org.openhab.binding.webexteams.internal.api.WebexTeamsApi;
+import org.openhab.binding.webexteams.internal.api.WebexTeamsApiException;
+import org.openhab.core.auth.client.oauth2.OAuthFactory;
+import org.openhab.core.config.core.Configuration;
+import org.openhab.core.library.types.DateTimeType;
+import org.openhab.core.library.types.StringType;
+import org.openhab.core.thing.ChannelUID;
+import org.openhab.core.thing.Thing;
+import org.openhab.core.thing.ThingStatus;
+import org.openhab.core.thing.ThingStatusDetail;
+import org.openhab.core.thing.binding.BaseThingHandler;
+import org.openhab.core.thing.binding.ThingHandlerService;
+import org.openhab.core.types.Command;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link WebexTeamsHandler} is responsible for handling commands, which are
+ * sent to one of the channels.
+ *
+ * @author Tom Deckers - Initial contribution
+ */
+@NonNullByDefault
+public class WebexTeamsHandler extends BaseThingHandler {
+
+    private final Logger logger = LoggerFactory.getLogger(WebexTeamsHandler.class);
+
+    private @Nullable WebexTeamsConfiguration config;
+
+    private final OAuthFactory oAuthFactory;
+    private final HttpClient httpClient;
+    private @Nullable WebexTeamsApi client;
+
+    private ThingStatus status = ThingStatus.UNKNOWN;
```

Comment:
Not need to define a member for the thing status.
You have already an access to this status with `getThing().getStatus()` and `updateStatus(...)`

Reply:
Removed member